### PR TITLE
Models 2

### PR DESCRIPTION
**Pull request by clair3st**
Added new models to imagersite for Photo and Ablums

### CODE REVIEW

File: imagersite/imagersite/settings.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+
+LOGIN_REDIRECT_URL = 'home'
+
+
+# Email setup
+ACCOUNT_ACTIVATION_DAYS = 7
+
+EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
+
```
Comment (Bl41r) [inline imagersite/imagersite/settings.py:139]: This is really cool that you have the email setup for your project going.

File: imagersite/imagersite/settings.py [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_PORT = 465
+EMAIL_USE_TLS = True
+
+EMAIL_HOST_USER = 'coding.projects17@gmail.com'
+EMAIL_HOST_PASSWORD = os.environ["EMAIL_PASSWORD"]
```
Comment (Bl41r) [inline imagersite/imagersite/settings.py]: Maybe use an os.environ.get here

File: imagersite/imager_images/models.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+# Create yo  models here.
+
+SHARING = (
+    ('PRIVATE', "Private"),
+    ('SHARED', "Shared"),
+    ('PUBLIC', "Public"),
+)
+
+
+class Photo(models.Model):
```
Comment (Bl41r) [inline imagersite/imager_images/models.py:17]: don't forget to use the @python_2_unicode_compatible decorators on all of your models

File: imagersite/imager_images/models.py [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    """One individaul photo."""
+
+    image_file = models.ImageField(upload_to="")
```
Comment (Bl41r) [inline imagersite/imager_images/models.py:20]: Idk if you care, but did you know that you can use patterns here?  Like upload_to='images/%Y-%m-%d'

File: imagersite/imager_images/models.py [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    title = models.CharField(max_length=255, blank=True, null=True)
+    description = models.CharField(max_length=255, blank=True, null=True)
+    date_uploaded = models.DateField(auto_now_add=True)
+    date_modified = models.DateField(blank=True, null=True)
+    date_published = models.DateField(blank=True, null=True)
+
+    photographer = models.ForeignKey(UserProfile,
+                                        related_name='photo',
+                                        blank=True,
```
Comment (Bl41r) [inline imagersite/imager_images/models.py]: I don't think you want blank=True here

File: imagersite/imager_images/models.py [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+                                 blank=True,
+                                 null=True)
+
+
+class Album(models.Model):
+    """Album class, one user, many photos."""
+
+    owner = models.ForeignKey(UserProfile,
+                              related_name="albums",
+                              blank=True,
```
Comment (Bl41r) [inline imagersite/imager_images/models.py]: same with this balnk

File: imagersite/imager_images/tests.py [hunk 7]
```diff
@@ -0,0 +1,242 @@
...
+    def test_album_has_an_owner(self):
+        """Test album has a photographer, which is tied to a user."""
+        this_user = self.users[0]
+        this_user.username = "User 1"
+        this_user.save()
+        this_album = self.albums[0]
+        this_album.owner = this_user.profile
+        this_album.save()
+        self.assertTrue(self.albums[0].owner.user.username == "User 1")
+
```
Comment (Bl41r) [inline imagersite/imager_images/tests.py:90]: Excellent job testing all of these attributes on the models.  I think to save yourself some time/space, you could make this a little easier by adding all of the attributes into a list of tuples, with an input value and the attribute, and you could just write one test to test them all using setattr, getattr, etc....  Of course, you wouldn't have that large number of tests that you used to have but...

Comment (clair3st) [reply imagersite/imager_images/tests.py:90]: Yes, we definitely could do this, in the future we will apply this method.

File: imagersite/imagersite/settings.py (hunk 1 above)
Comment (clair3st) [reply imagersite/imagersite/settings.py:139]: We are legit

**Pull request merged**